server/views/pages/manageConditions/curfew/input.njk

```diff
@@ -45,7 +49,132 @@
                         <div class="govuk-hint">{{ config.subtext }}</div>
                     {% endif %}
 
-                    {{ formBuilder(licence.id, config, additionalCondition, validationErrors, formResponses, csrfToken) }}
+                    {% macro renderTimePicker(id, labelText, formResponses) %}
+                        {{ timePicker({
+                            name: id,
+                            id: id,
+                            label: {
+                            text: labelText,
+                            classes: "govuk-!-font-weight-bold"
+                            },
+                            hint: {
+                            text: "For example, 9:30am or 2:55pm"
+                            },
+                            errorMessage: validationErrors | findError(id),
+                            formResponses: formResponses[id] or curfewTimes[id]
+                        }) }}
+                    {% endmacro %}
+
+
+                    {% set oneCurfewHtml %}
+                        {{ renderTimePicker("oneCurfewStart", "Enter the curfew start time", formResponses) }}
+                        {{ renderTimePicker("oneCurfewEnd", "Enter the curfew end time", formResponses) }}
+                    {% endset %}
+
+                    {% set twoCurfewsHtml %}
+                        {{ renderTimePicker("twoCurfewStart", "First curfew – enter the start time", formResponses) }}
+                        {{ renderTimePicker("twoCurfewEnd", "First curfew – enter the end time", formResponses) }}
+                        {{ renderTimePicker("twoCurfewStart2", "Second curfew – enter the start time", formResponses) }}
+                        {{ renderTimePicker("twoCurfewEnd2", "Second curfew – enter the end time", formResponses) }}
+                    {% endset %}
+
+                    {% set threeCurfewsHtml %}
+                        {{ renderTimePicker("threeCurfewStart", "First curfew – enter the start time", formResponses) }}
+                        {{ renderTimePicker("threeCurfewEnd", "First curfew – enter the end time", formResponses) }}
+                        {{ renderTimePicker("threeCurfewStart2", "Second curfew – enter the start time", formResponses) }}
+                        {{ renderTimePicker("threeCurfewEnd2", "Second curfew – enter the end time", formResponses) }}
+                        {{ renderTimePicker("threeCurfewStart3", "Third curfew – enter the start time", formResponses) }}
+                        {{ renderTimePicker("threeCurfewEnd3", "Third curfew – enter the end time", formResponses) }}
```

Comment:
mmh.. is there a reason why we're rendering so many versions of this? 

could we have a single macro `renderCurfews` that takes a numeric argument about how many curfews to render? 

Then we shouldn't need to have 6 different curfew fields to read from in the validator?